Seminars/Sem.02/Pract.02/Task04.cpp

```diff
@@ -0,0 +1,128 @@
+#include <iostream>
+#include <fstream>
+
+const char RELATION_PATH[] =
+        "C:\\Users\\nikol\\Documents\\"
+        "GitHub\\Object-oriented_programming_FMI_2023\\"
+        "Seminars\\Sem.02\\Pract.02\\"
+        "relation.txt";
+
```

Comment:
What is the difference between absolute and relative path?

Reply:
Relative path works only in current directory 
Absolute come from root

I use Absolute because relative don't work 
I don't know why, maybe program searching it where .exe runs